### PR TITLE
feat(stubs): do not use spies, give example for stubs.

### LINKED ISSUE #7: spies

### PR DESCRIPTION
**Pull request by tykowale**
Closes #7

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -316,13 +316,18 @@ describe('something', function () {
...
 
 ## Sinon
-### Use Sinon stubs
+### [Use Sinon stubs](http://sinonjs.org/docs/#stubs)
 
 Stubs allow observation of existing methods on fixtures and forcing the code down different paths to test all branches.
 
 Mocks have been deprecated and will not be supported in `sinon@2`.
 
+Spies are useful for watching functions, but `sinon.stub` has all of the functionality and are more hands on.
```
Comment (alexlrobertson) [inline README.md]: "are more hands on" => "is more hands on"

File: README.md [hunk 2]
```diff
@@ -334,6 +339,46 @@ var contextService = {
 };
 ```
 
+#### Example
+
+In this example we have a inject the service `upUserService` which has a method `getCurrentUser` into our `LogOutController`. In this example we are testing a controller, so we want to mock out the service functionality.
```
Comment (alexlrobertson) [inline README.md]: "we have a inject the service" => "we have an injection of the service"?

**Comment by alexlrobertson** [comment]
Show some stub expectations maybe?

File: README.md (hunk 1 above)
Comment (tykowale) [inline README.md]: This was changed in the newest commit but for some reason isn't updating here

**Comment by alexlrobertson** [comment]
:eggplant: & :cake:

**Pull request merged**